python_backend/api/v1/endpoints/documents.py

```
@@ -22,6 +23,54 @@ def _get_documents_path() -> Path:
+@router.delete("/{document_name:path}", response_model=Dict[str, str])
+async def delete_document(document_name: str) -> Dict[str, str]:
+    """
+    지정된 문서를 삭제합니다.
+    - document_name에 하위 디렉토리 경로를 포함할 수 있습니다. (예: subdir/doc.pdf)
+    """
+    try:
+        base_dir = _get_documents_path()
+        file_path = base_dir / document_name
+
+        if not file_path.is_file():
+            raise HTTPException(status_code=404, detail=f"문서를 찾을 수 없습니다: {document_name}")
```

Ruff (0.13.1)

Abstract `raise` to an inner function

(TRY301)